yt_dlp/extractor/tiktok.py

```diff
@@ -30,14 +32,37 @@
 
 
 class TikTokBaseIE(InfoExtractor):
-    _APP_VERSIONS = [('26.1.3', '260103'), ('26.1.2', '260102'), ('26.1.1', '260101'), ('25.6.2', '250602')]
-    _WORKING_APP_VERSION = None
-    _APP_NAME = 'trill'
-    _AID = 1180
+    _APP_INSTALL_IDS = [
```

Comment:
I assume we know the owner of these ids? We don't wanna use random people's id without their permission.

Reply:
yep they were donated directly